docs/user/production-considerations/alerting-production-considerations.asciidoc

```diff
@@ -38,7 +38,7 @@ For detailed guidance, see <<alerting-troubleshooting, Alerting Troubleshooting>
 [[alerting-scaling-guidance]]
 === Scaling guidance
 
-As rules and actions leverage background tasks to perform the majority of work, scaling Alerting is possible by following the <<task-manager-scaling-guidance,Task Manager Scaling Guidance>>.
+As rules and actions leverage background tasks to perform the majority of work, scaling alerting is possible by following the <<task-manager-scaling-guidance,Task Manager Scaling Guidance>>.
```

Comment:
Hmm. This sounds like maybe it should read as `Kibana Alerting` as we're talking about scaling our specific framework

Reply:
Updated in https://github.com/elastic/kibana/pull/119314/commits/f8dcbce022353d226332a67391e98c9d37159051